### ISSUE TITLE
undefined method `class_inheritable_accessor'

### ISSUE THREAD

**Issue by cmezak** [jpablobr/active_paypal_adaptive_payment#13]
Getting this error. Probably a rails 3.2 compatibility issue. Will look into it.

**Comment by ralphos** [comment]
Any luck finding a solution to this? I'm also trying to get it to work with a Rails 3.2.1 app and getting the same error.

**Comment by jpablobr** [comment]
Haven't been able to reproduce the error :/ However, I noticed @cmezak modified the gemspec activemerchant version dependency on his fork:

https://github.com/cmezak/active_paypal_adaptive_payment/commit/c6661704c45b73a0be5c608c673b106ac1eb5b69

Can you try that?

This might be related to [issue 11](https://github.com/jpablobr/active_paypal_adaptive_payment/issues/11)  which was solved with [this](https://github.com/jpablobr/active_paypal_adaptive_payment/commit/6b97577ef119e67164d1689c090c9f22928c6d0b) commit.

I'll try to get into it over the weekend.

**Comment by cmezak** [comment]
My fork seems to have fixed it. Now I'm just on to other problems. Paypal's API is ridiculously complicated!

**Comment by jpablobr** [comment]
@cmezak yeah, I think `>= 1.5.1` is the way to go, `~> 1.5.1` is too restrictive. It will not allow anything > 1.5 such as 1.6 :/ and I guess you guys on Rails 3.2 might be on active_merchant 1.2+. Can you send me a pull request with your changes so I can merge them in? :)

**Comment by ralphos** [comment]
Thanks guys. Forked my first gem and it worked :) Happiness.

**Comment by ralphos** [comment]
As mentioned earlier, I'm new to Rails. My fork worked but now I'm getting `uninitialized constant ApplicationController::Activemerchant` from my 'gateway' method as per the documentation. My understanding is I don't have to include Activemerchant in my gemfile as it's already included with active_paypal_adaptive_payment? I tried including it anyway but still no luck. Any idea why this might be happening? I've tried putting require 'activemerchant' in the application controller. Appreciate any help. Thx!
